db/models.py

```diff
@@ -46,3 +54,77 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Change "User" to "settings.AUTH_USER_MODEL" - in a video Dania sad, that this is the good practice.